eurocodedesign/units/__init__.py

```diff
@@ -0,0 +1,255 @@
+        if isinstance(other, (int, float)):
+            return type(self)(self._value
+                              / other
+                              / (self._prefix.value
+                                 ** self._power), self._prefix)
+        self_type = type(self)
+        other_type = type(other)
+
+        if self_type == other_type:
+            return float(self._value/other._value)
+
+        if self_type not in _allowed_multiplications.values():
+            raise TypeError('No divisions by another unit allowed'
+                            f' for f{self_type}')
+        keys = [k for k, v in _allowed_multiplications.items()
+                if v == self_type]
+        for type_pair in keys:
+            if other_type in type_pair:
+                l_ = list(type_pair)
+                l_.remove(other_type)
+                new_type = l_[0]
+                return new_type(self._value / other._value)
+        raise TypeError(f'Division of {self_type} by '
+                        f'{other_type} not allowed.')
+
+    def __lt__(self, other: Self):
+        if (type(self) != type(other) or
+            self._physical_type != other._physical_type):
+            raise TypeError
+        return self._value < other._value
+
+    def __le__(self, other: Self):
+        if (type(self) != type(other) or
+            self._physical_type != other._physical_type):
+            raise TypeError
+        return self._value <= other._value
+
+    def __eq__(self, other: Self):
+        if (type(self) != type(other) or
+            self._physical_type != other._physical_type):
+            raise TypeError
+        return self._value == other._value
+
+    def __ne__(self, other: Self):
+        if (type(self) != type(other) or
+            self._physical_type != other._physical_type):
+            raise TypeError
+        return not (self == other)
+
+    def __ge__(self, other: Self):
+        if (type(self) != type(other) or
+            self._physical_type != other._physical_type):
+            raise TypeError
+        return self._value >= other._value
+
+    def __gt__(self, other: Self):
+        if (type(self) != type(other) or
+            self._physical_type != other._physical_type):
+            raise TypeError
+        return self._value > other._value
+
+    def to(self, prefix: Prefix):
+        """
+        Convert unit prefix to given prefix
+        @param prefix: Prefix,
+        @return: None
+        """
+        if not self._is_prefix_allowed(prefix):
+            raise TypeError(f"Prefix {prefix} not allowed for {type(self)}")
+        self._prefix = prefix
+        return
+
+
+class Seconds(AbstractUnit):
+    _physical_type = PhysicalType.TIME
+
+
+class Meter(AbstractUnit):
+    _physical_type = PhysicalType.LENGTH
+
+
+class Meter_2(AbstractUnit):
+    _power = 2
+    _physical_type = PhysicalType.AREA
+
+
+class Meter_3(AbstractUnit):
+    _power = 3
+    _physical_type = PhysicalType.VOLUME
```

Comment:
m3 does not always correspond to volume - e.g. the plastic and elastic section moduli and the static moment also have units in m3.

We should discuss.